app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,11 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+
+    <!--    my colors-->
+    <color name="my_white">#ececec</color>
+    <color name="my_light_gray">#D8D8D8</color>
+    <color name="my_dark_gray">#6D6D80</color>
+    <color name="my_pink">#FF3466</color>
+    <color name="my_dark_blue_background">#191926</color>
```

Comment:
try to give them another names. For example you can just remove "my_" and it would be better